include/eve/detail/function/simd/x86/load.hpp

```diff
@@ -90,7 +90,7 @@ namespace eve::detail
   //================================================================================================
   // logical loads require some specific setup
   //================================================================================================
-  template<typename T, typename N, typename Ptr, x86_abi ABI>
+  template<typename T, typename N, typename Ptr, typename ABI>
```

Comment:
if this is no more x86 related, it should be moved to the detail/function/simd/common/load.hpp file

Reply:
But it is x86 specific? I mean on ARM/POWER we should just load an not do any conversions.

Reply:
OK, there is an issue. Fixing outside